ibus/LICENSE

```diff
@@ -0,0 +1,350 @@
+This source code on this software based on ibus-hangul engine code.
+We gratefully acknowledge these and other contributions to libhanjp.
+We recommend that programs that use libhanjp also acknowledge these contributions,
+and quote these license statements in an appendix to the documentation.
+
+ibus-hangul
+https://github.com/phuang/ibus-hangul
+GNU General Public License v2.0
+Huang Peng <shawn.p.huang@gmail.com>
+Choe Hwanjin <choe.hwanjin@gmail.com>
```

Comment:
Rather then putting acknowledgement on `LICENSE` file, consider putting this on `README.md`. As `README.md` is more easily found by other visitors.
AFAIK, `LICENSE` usually only contains full license text. And other thing like acknowledgements are usually included in `README.md` or other separate file such as `THANKS` file.

Reply:
Right, I've got missed.
I'll fix for it.